### ISSUE TITLE
Add Microsoft Azure support

### ISSUE THREAD

**Issue by chuwy** [snowplow-archive/snowplow-snowflake-loader#47]
This obviously needs to be split into separate tickets, but leaving for ideas. Now it is officital: Snowflake [supports MS Azure](https://www.snowflake.net/global-snowflake-loading-data-into-snowflake-from-azure-blob-storage/)

**Comment by alexanderdean** [comment]
Makes sense - an Azure-idiomatic manifest back-end would go well with this...

**Comment by alexanderdean** [comment]
Also need to come up with somewhere to run the Snowflake Transformer
